content/4_Gold/Combinatorics.problems.json

```diff
@@ -118,6 +118,18 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "ac-XorPyramid",
```

Comment:
wait is pascalcase how we usually name these

Reply:
Yep that's how the majority of ac problem are named